src/test/java/com/hocs/test/pages/decs/BasePage.java

```diff
@@ -32,6 +32,10 @@
 
     public static String currentPlatform = "";
 
+    public static String newCaseRef = "";
+
+    public static String oldCaseRef = "";
+
```

Comment:
As you are only using these 2 string in a method where you use them to set a new session variable, I think they should only be scoped (instantiated) within the method, and not as page object variables.